frontend/app/sysupport/index/route.js

```diff
@@ -0,0 +1,8 @@
+import Route from '@ember/routing/route'
+
+export default Route.extend({
+  model() {
+    this.set('breadCrumb', { title: 'Overview' })
```

Comment:
This is not supposed to be here since it's static.. Also, it is not translated!

Reply:
:+1: